### DISCUSSION TITLE
Private information in the suggestion

### DISCUSSION THREAD

**Discussion by butcher51** [community/community#46009] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I guess this information is too sensitive:
![james](https://user-images.githubusercontent.com/4325739/216036224-1d29a06f-1af1-4748-b63f-8cd4606b2015.png)

**Comment by Akash1134** [comment]
Hi @butcher51 ,

We follow responsible practices in accordance with our [Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement) to ensure that your code snippets will not be used as suggested code for other users of GitHub Copilot.

For more information on how GitHub processes and uses personal data, please see the [GitHub Copilot Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/).
